haxepunk/graphics/atlas/TileAtlas.hx

```diff
@@ -28,10 +33,29 @@ class TileAtlas extends Atlas
 		{
 			throw 'Atlas doesn\'t have a region number "$index"';
 		}
-		
+
 		return _regions[index];
 	}
-	
+
+	/**
+	 * Loads a TexturePacker xml file and generates all tile regions.
+	 * Uses the Generic XML exporter format from Texture Packer.
+	 * @param	file	The TexturePacker file to load
+	 * @param	sprites	A list of the sprite names in the packed file
+	 * @return	A TileAtlas with all packed images defined as regions ordered by sprite names supplied
+	 */
+	public static function loadTexturePacker(file:String, sprites:Array<String>):TileAtlas
```

Comment:
That's a good point. I'm not sure what the appropriate name would be for this function.